CONTRIBUTING.md

```diff
@@ -1,44 +1,72 @@
 # Contributing
 
-If you're interested in joining the community to build a better SQL, there are
-lots of ways of contributing - big and small:
+If you're interested in joining the community to build a better SQL, here is how you start:
```

Comment:
If it's OK, let me do a pass of this — I found the existing sentence more welcoming and playful! (Overall the changes are great, and completely the right impulse to make changes — I'm commenting to check it's OK to iterate once more)

Reply:
Sure, this was my intention. 

The main driving point was to segment the areas so new contributors can pick easily, instead of reading all directions for. Also links to headers.

Reply:
For sure for sure, merge away and I'll take a quick pass. (overall it's a clear improvement!)

Reply:
This is not done, so I won't merge it now. Feel free to push to this branch.